app/models/course.rb

```diff
@@ -53,10 +53,12 @@ class Course < ApplicationRecord
   has_many :site_statuses
   has_many :sites, through: :site_statuses
 
-  scope :changed_since, ->(datetime, from_course_id = 0) do
-    if datetime.present?
-      where("course.updated_at >= ? AND course.id > ?", datetime, from_course_id).order(:updated_at, :id)
-    end
+  scope :changed_since, ->(timestamp) do
+    if timestamp.present?
+      where("course.updated_at > ?", timestamp)
+    else
+      where("updated_at is not null")
```

Comment:
would `where.not(updated_at: nil)` be more idiomatic?

Reply:
done, tests still pass, cheers